src/tensor.c

```diff
@@ -541,22 +541,9 @@ enum xnn_shape_inference_status xnn_tensor_propagate_dimension(
     return xnn_shape_inference_status_no_change;
   }
 
-  if (inferred_dim < to->shape.minimum_dim[to_dim]) {
```

Comment:
is there a use for `minimum_dim` field in `xnn_shape` going forward?

Reply:
Again another artefact which can probably go